### ISSUE TITLE
[Icon Request] Support for Latino

### ISSUE THREAD

**Issue by MelvinG24** [vscode-icons/vscode-icons#2452]
- [ x] I'm sure this issue is not a _duplicate_.
- [ x] I'm submitting an icon request:
  - Type: `language id`
  - Icon Name: `latino`
  - Sample original Icon: ![logo](https://raw.githubusercontent.com/MelvinG24/Latino-VSCode/master/images/iconsFiles/icons/Lenguaje_Latino-Icon.svg?sanitize=true)
  - Language ids: `latino`

- Language Extensions: https://marketplace.visualstudio.com/items?itemName=MelvinGuerrero.latino-vscode
- More info: https://github.com/MelvinG24/Latino-VSCode

**Comment by KingDarBoja** [comment]
This icon is beautiful without mention it resembles some latin american glyphs.

**Comment by JimiC** [comment]
Same impression :smile:

**Comment by KingDarBoja** [comment]
The icon is already done but unfortunately its size is around 10 KB.

**Comment by JimiC** [comment]
Figures with so many details. We need to reduce its size.

**Comment by KingDarBoja** [comment]
Managed to simplify a bit and reduce its size to 3 KB, not proud of what I did but it works.

![latino-icon-low-res](https://user-images.githubusercontent.com/30560560/79075835-cf2fd080-7cba-11ea-9607-db32d8fad883.png)

**Comment by JimiC** [comment]
Keep the different color

**Comment by JimiC** [comment]
I'll have a try on it also

**Comment by KingDarBoja** [comment]
Did a quick try on the original and got a better result and same size (3.03 KB, 4 KB on disk).

![new-latino-low-res](https://user-images.githubusercontent.com/30560560/79075947-a3f9b100-7cbb-11ea-9541-eb12646a4c4f.png)

**Comment by JimiC** [comment]
That would have been my approach. I would keep the hole for the eye though.

**Comment by MelvinG24** [comment]
Hi!
I am graphic designer as well. If you need me to make the file minimalist or smaller in size please, let me know so I can help. Is their any specify size for the icon?

Muchas Gracias for all the support on my request I really appreciated it

**Comment by MelvinG24** [comment]
> The icon is already done but unfortunately its size is around 10 KB.

> Figures with so many details. We need to reduce its size.

Hello again,
I was comparing my icon with some of the icons inside the extension program file and I got a littler confused. I found icons more complex in details and bigger in size such as **file_type_light_lerna**. My question is, is their any specific reason why the icon have to be smaller in size?

**Comment by JimiC** [comment]
Icons that refer to filenames can be bigger than 3kb simply because they are used only once in the editor.

**Comment by MelvinG24** [comment]
Hi,
I make a new svg logo about 5KB, please, let me know if this one work for you.

Thank you
[Latino-Logo-New.zip](https://github.com/vscode-icons/vscode-icons/files/4478200/Latino-Logo-New.zip)

**Comment by KingDarBoja** [comment]
I don't know what you did (maybe reduce the number of path joints???) but this is great!

**Comment by JimiC** [comment]
Yes reducing the joints is the only way but still not enough. I'm working on it too.

**Comment by MelvinG24** [comment]
> Yes reducing the joints is the only way but still not enough. I'm working on it too.

Hi guys,
I just want to say that I finally made it. This new file have 2KB. I redraw the logo trying to minimize the amount of point and it works. Here is the new logo.

Thank you again for you collaboration

Logo: 
[Lenguaje_Latino-Icon.zip](https://github.com/vscode-icons/vscode-icons/files/4485852/Lenguaje_Latino-Icon.zip)

**Comment by JimiC** [comment]
Absolutamente perfecto :tada:

**Comment by MelvinG24** [comment]
Hi again!
I saw you already merge the pull request on the code, I can not be more excited about it. However, this morning created another new file and this one is about 1KB. I just want to share it with you guys in case you think this new one will preform better than the last one.

Thank you again for your support
New Logo 1KB:
[Lenguaje_Latino-Icon.zip](https://github.com/vscode-icons/vscode-icons/files/4489998/Lenguaje_Latino-Icon.zip)

**Comment by JimiC** [comment]
I'll make an update PR.

**Comment by Alhadis** [comment]
@MelvinG24 Oh good, I'm not the only one who enjoys golfing SVGs down to the fewest points possible. 😀 

Finally, an icon I don't have to optimise myself. 😉

SALU2!
